documentation/shipit.yml.md

```diff
@@ -0,0 +1,118 @@
+## Configuring shipit.yml
+
+All the settings in `shipit.yml` are totally optional. Most applications can be deployed without any configuration.
+
+The `shipit.yml` file has X main sections:
+
+review
+  checklist
+  monitoring
+fetch
+deploy
+rollback
+
+tasks
+  restart
+  unlock
+
+dependencies
+  bundler
+
+### review
+
+<code>review.checklist</code>
+
+Array of strings. Can contain and HTML tags. Users are required to check each of them before being able to deploy.
+
+
+<code>review.monitoring</code>
+
+List of inclusions. Inclusions can either be of type image or iframe:
+
+```
+review:
+  monitoring:
+    - image: https://example.com/monitoring.png
+    - iframe: https://example.com/monitoring.html
+```
+
+
+### fetch
+
+<code>fetch</code>
+
+Array of shell commands, shipit will execute them regularly to check the deployed revision.
+
+foo # command to run to get the revision of the currently-deployed version, defaults to disabled
+
+curl --silent https://app.shopify.com/services/ping/version
+
+### deploy
+
+<code>deploy.override</code>
+
+Array of shell commands required to deploy the application. Shipit will try to infer it from the repository structure, but you can change the default inference.
+
+exec ./script/multi_deploy.rb bundle exec cap $ENVIRONMENT deploy

```

Comment:
No it's a custom one Rory grabbed from one of our repos. The default inference is based on multiple heuristics: https://github.com/Shopify/shipit-engine/tree/master/app/models/deploy_spec